lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingLink.cs

```diff
@@ -57,16 +57,19 @@ internal sealed class StreamingLink<T> : ILink<T>
         /// </summary>
         /// <param name="remoteEndpoint">The remote endpoint to connect to</param>
         /// <param name="streamingCodec">Streaming codec</param>
-        internal StreamingLink(IPEndPoint remoteEndpoint, IStreamingCodec<T> streamingCodec)
+        /// <param name="connectionRetries">Number of re-tries to connect to remote endpoint</param>
+        /// <param name="sleepTime">Sleep time in milliseconds between retries</param>
+        internal StreamingLink(IPEndPoint remoteEndpoint,
+            IStreamingCodec<T> streamingCodec,
+            int connectionRetries = 20,
+            int sleepTime = 1000)
```

Comment:
Why are we splitting default in two places? The parameter definition already has defaults. I am worried of the effect of changing just one location.